.gitignore

```diff
@@ -1 +1 @@
-node_modules
+node_modules
```

Comment:
BTW take a note of the red mark github has added to the code
![image](https://user-images.githubusercontent.com/8686631/68088501-9583af80-fe68-11e9-86af-63bb2dd3fc40.png)

There is a convention that text files should end with an newline character (empty line). There are reasons behind (see e.g. this [explanation](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline)).

You do not have to add final newlines yourself. Make your IDE/editor do this for you. E.g. Sublime has a setting `ensure_newline_at_eof_on_save`. Just assign it `true`. [Sublime settings unleashed](http://docs.sublimetext.info/en/latest/reference/settings.html).

Reply:
thx!